ddtrace/contrib/asyncio/__init__.py

```diff
@@ -47,8 +51,13 @@
 with require_modules(required_modules) as missing_modules:
     if not missing_modules:
         from .provider import AsyncioContextProvider
+        from ...internal.context_manager import CONTEXTVARS_IS_AVAILABLE
+        from ...provider import DefaultContextProvider
 
-        context_provider = AsyncioContextProvider()
+        if not CONTEXTVARS_IS_AVAILABLE:
```

Comment:
Can you swap these cases around? It'll read easier if the `if` condition is the positive.

e.g.

```python
if CONTEXTVARS_IS_AVAILABLE:
    context_provider = DefaultContextProvider()
else:
    context_provider = AsyncioContextProvider()
```